### DISCUSSION TITLE
Use skip singnal to skip a pipeline.

### DISCUSSION THREAD

**Discussion by nirenyang** [comfyanonymous/ComfyUI#3495] [Ideas]
Dear developers, hello! 

We already have some plugins that provide workflow selection, such as Easy Use and "... switch node" in Chaosaiart, 

but they are all pre data selections. However, in practical use, people also have the need to choose the post work pipeline, 

because this can achieve more convenient selection and debugging by switching indices.

I hope these two pictures can illustrate this idea.

![skip1](https://github.com/comfyanonymous/ComfyUI/assets/1759638/8601d799-b4fa-4187-af94-45f75acd739a)

![skip0](https://github.com/comfyanonymous/ComfyUI/assets/1759638/032c6c75-a22b-4475-b089-36d77ce228e4)

Currently, writing this way may cause errors in subsequent nodes, but it is really an interesting idea. 
``` python
def node (self, source, condition_index): 
  if condition_index==1: 
    return ([], source) 
  else: 
    return (source, []) 
``` 

```  python
def node (self, source, condition_index): 
  if condition_index==1: 
    return (None, source) 
  else: 
    return (source, None) 
``` 

How should we implement this feature? 

Best regards!

**Comment by ltdrdata** [comment]
To handle this feature. 
This PR must be merged https://github.com/comfyanonymous/ComfyUI/pull/2666

The PR in question has a wide-ranging impact on ComfyUI, so it requires ample verification time. Currently, the delay is due to comfy being busy with SD3-related work.
